index.js

```diff
@@ -175,6 +179,18 @@ function tcurl(opts) {
             request.headers.as = 'raw';
             request.send(opts.endpoint, opts.head, opts.body,
                 onResponse);
+        } else if (opts.http) {
+            var ashttp = TCurlAsHttp({
+                remoteHostPort: opts.hostname + ':' + opts.port,
+                serviceName: opts.service,
+                method: opts.http,
+                path: opts.endpoint,
+                headers: JSON.parse(opts.head),
+                body: JSON.parse(opts.body),
```

Comment:
opts.head and opts.body are from command line options. JSON is just a way to allow specify cmd line inputs as objects.